landlab/components/hand_calculator/hand_calculator.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+"""Landlab component to calculate height above nearest drainage."""
+from warnings import warn
+
+import numpy as np
+
+from landlab import Component
+
+
+class HeightAboveDrainage(Component):
```

Comment:
Usually the convention is to have class and component names fit in the syntax of "X is a Y", for example: "is a string," "is an array," "is a RasterModelGrid," "is a LinearDiffuser". Consider modifying the name.

Reply:
Changed to HeightAboveDrainageCalculator, more in line with the HackCalculator.